torch/utils/data/dataloader.py

```diff
@@ -766,13 +773,14 @@ class DataLoader(object):
     __initialized = False
 
     def __init__(self, dataset, batch_size=1, shuffle=False, sampler=None, batch_sampler=None,
-                 num_workers=0, collate_fn=default_collate, pin_memory=False, drop_last=False,
-                 timeout=0, worker_init_fn=None):
+                 num_workers=0, collate_fn=default_collate, pin_memory=False, pin_fn=pin_memory_batch,
```

Comment:
Unfortunately adding an argument in the middle of the signature is not a backward compatible change 😕

Reply:
i'm not sure we need this change. the data loader has so many arguments, no one uses it as *args, only as kwargs